src/core/smallvec_type.hpp

```diff
@@ -150,12 +119,7 @@ class SmallVector {
 	 */
 	inline void Resize(uint num_items)
 	{
-		this->items = num_items;
-
-		if (this->items > this->capacity) {
-			this->capacity = Align(this->items, S);
-			this->data = ReallocT(this->data, this->capacity);
-		}
+		std::vector<T>::resize(num_items);
```

Comment:
Do we even need Resize()? I'm sure it's just an implementation detail due to creating the data array ourselves.